charts/vh-booking-queue-subscriber/values.dev.template.yaml

```diff
@@ -0,0 +1,8 @@
+---
+function:
+  image: '${IMAGE_NAME}'
+  triggers:
+  - type: azure-servicebus 
+    serviceBusName: "vh-infra-core-demo"
+    queueName: booking
+    messageCount: 1
```

Comment:
these were not in the flux change. do they need to be added in there? and do the messageCount need to be different per environment?

Reply:
message count is the number of messages in the queue before a job starts, so for our use case, we would never want it to be higher than 1, for example, because messages are only put on the queue when a admin creates a hearing in admin web, if only one was created all day, there would be a delay until there is a total of 5 on the queue creating a massive delay.

not sure how many hearings are made a day, but cannot imagine there being loads.